Makefile

```diff
@@ -198,8 +198,8 @@ update-version:
 	$(SED) '1s/.*/VERSION=$(VERSION)/' access/email/Makefile
 	$(SED) '1s/.*/VERSION=$(VERSION)/' terraform/install.mk
 	$(MAKE) update-helm-version
-	$(MAKE) terraform-gen-tfschema
 	$(MAKE) update-teleport-dep-version
+	$(MAKE) terraform-gen-tfschema
```

Comment:
@r0mant I had to change the order so that we generate the tf schema for the version we are about to release.
The previous order meant that we were releasing the previous version's fields.
Eg, when releasing 13.0.3 we would be using the teleport/api 13.0.2

With this change we should get the latest schema when doing a release.
So, when you release 13.0.4, we should get the terraform schema of teleport/api 13.0.4.